google/resource_compute_instance.go

```diff
@@ -328,6 +311,14 @@ func resourceComputeInstance() *schema.Resource {
 							Computed: true,
 						},
 
+						"network_ip": &schema.Schema{
```

Comment:
@danawillow what was the plan with `address` and `network_ip`?

Reply:
Looks like @negz put some info about this in the PR description

Reply:
My bad, it makes total sense to mark it as deprecated.